back/spec/models/app_configuration/settings_spec.rb

```diff
@@ -59,7 +60,10 @@ def self.feature_description; 'Oh my... such a good feature.' end
       let(:json_schema) { described_class.json_schema }
 
       it 'includes core json schema settings in schema' do
-        expect(json_schema).to include(described_class.core_settings_json_schema)
```

Comment:
Previously, `core_settings_json_schema` included `dummy_feature` because of lack of `deep_dup` in `json_schema`